vhost-user-backend/src/event_loop.rs

```diff
@@ -57,6 +58,44 @@ impl std::error::Error for VringPollError {}
 /// Result of vring epoll operations.
 pub type VringEpollResult<T> = std::result::Result<T, VringPollError>;
 
+bitflags! {
+    #[derive(Debug, PartialEq, PartialOrd)]
+    pub struct VringPollEvent: u32 {
+        const READABLE = 1u32;
+        const WRITABLE = 2u32;
+    }
+}
+
+impl From<Interest> for VringPollEvent {
+    fn from(value: Interest) -> Self {
+        let mut event = VringPollEvent::empty();
+        if value.is_readable() {
+            event |= VringPollEvent::READABLE;
+        }
+        if value.is_writable() {
+            event |= VringPollEvent::WRITABLE;
+        }
+        return event;
+    }
+}
+
+impl From<VringPollEvent> for Interest {
+    fn from(value: VringPollEvent) -> Self {
+        let mut interest = None;
+        if value == VringPollEvent::READABLE {
+            interest = interest
+                .map(|interest| Interest::READABLE | interest)
+                .or(Some(Interest::READABLE));
+        }
+        if value == VringPollEvent::WRITABLE {
+            interest = interest
+                .map(|interest| Interest::WRITABLE | interest)
+                .or(Some(Interest::WRITABLE));
+        }
+        return interest.expect("Interest is invalid");
+    }
+}
```

Comment:
Where do we use this?, I mean, why we need to convert from Interest to VringPollEvent?, if we don't do it, better removing it, remember we want to hide mio::Interest